### PR TITLE
fix lbt mods

### PR DESCRIPTION
**Pull request by feigewang**
fix LBT MODS blue side. 
Per request from our users, I would like to get this merged into develop as soon.

### CODE REVIEW

**Review by profxj — CHANGES_REQUESTED** [review]
pass tests (unit + DevSuite on LBT only) and I'll approve.

Thanks!

**Review by tepickering — APPROVED** [review]
LGTM! i am not MODS-savvy so i can't speak to the specific changes to the parameters, but the code changes look straightforward and self-contained.

**Comment by tepickering** [comment]
looks like there's a test failure against `astropy-master`, but it's not relevant to this work....

File: pypeit/spectrographs/lbt_mods.py
Comment (jhennawi) [inline pypeit/spectrographs/lbt_mods.py:434]: How was this ever working before if the wavelength axis was flipped. I don't see how you could have gotten a wavelength solution out of full_template with flipped wavelength direction.

**Comment by profxj** [comment]
@tepickering -- is it possible to initiate the CI tests on the `develop` branch?
i.e. to see if that is now failing (I presume it).

File: pypeit/spectrographs/lbt_mods.py
Comment (jhennawi) [reply pypeit/spectrographs/lbt_mods.py:434]: Ah okay, now I see you checked in a new arxiv, so I guess you flipped it.

**Review by jhennawi — APPROVED** [review]
Approved. I presume this spectrograph is in the dev suite? Please run just that one test before you merge.

**Comment by tepickering** [comment]
i haven't seen this test failure in my branch which is pretty closely synced to `develop`. digging a little deeper. working on setting up scheduled workflows that check `master` weekly and `develop` daily to help flag when upstream changes might bite us.

**Comment by profxj** [comment]
Please do, @tepickering .  Thanks!

**Review by profxj — APPROVED** [review]
approving now

**Comment by tepickering** [comment]
i can't reproduce that `astropydev` failure locally. also didn't show up in the last CI run against my fork yesterday which was up-to-date with `develop`. will revisit tomorrow when i set up my PR.

**Comment by feigewang** [comment]
Both tests passed. Note that the failure on the test_scripts are caused by my Ginga problem. 

Screen Shot 2021-01-19 at 7 08 29 PM
Screen Shot 2021-01-19 at 6 44 03 PM

**Pull request merged**
